README.md

```diff
@@ -1,3 +1,12 @@
 Some OQL scripts can be used to analyze the heapdumps of Elasticsearch intances with VisualVM.
 
-- [query.js](query.js): extracts Elasticsearch queries in the heapdump
\ No newline at end of file
+- [query.js](query.js): extracts Elasticsearch queries in the heapdump
+
+Instructions
```

Comment:
❤️

Reply:
Thank you for putting this together, it has been super useful